src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -104,7 +112,7 @@ public function __construct(Table $table, array $config = [])
      */
     public function initialize(array $config)
     {
```

Comment:
Since the locator is used multiple times in the same method, it would be nice if you could copy it in a local variable